third_party/packages/cupertino_icons/pubspec.yaml

```diff
@@ -3,11 +3,19 @@ name: cupertino_icons
 description: Default icons asset for Cupertino widgets based on Apple styled icons
 repository: https://github.com/flutter/packages/tree/main/third_party/packages/cupertino_icons
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+cupertino_icons%22
-version: 1.0.6
+version: 1.0.7
 
 environment:
   sdk: ">=3.0.0 <4.0.0"
 
+dependencies:
+  flutter:
```

Comment:
I have moved this to dev-dependency.

Reply:
Looks like this will need a minor change to the repo tooling, so that it uses `flutter test` when there's a dev dependency on `flutter`. I'll make that change this week.